common/lib/capa/capa/capa_problem.py

```diff
@@ -822,22 +822,17 @@ def _extract_html(self, problemtree, question_id=0):  # private
         # otherwise, render children recursively, and copy over attributes
         tree = etree.Element(problemtree.tag)
         for item in problemtree:
-            item_xhtml = self._extract_html(item, question_id)
-
-            if item.tag == 'question':
-                item_xhtml.set('class', 'question')
-                item_xhtml.set('id', 'question-{}'.format(question_id))
-                question_id += 1
+            item_xhtml = self._extract_html(item)
 
             if item_xhtml is not None:
                 tree.append(item_xhtml)
 
         if tree.tag in html_transforms:
             tree.tag = html_transforms[problemtree.tag]['tag']
-        else:
-            # copy attributes over if not innocufying
-            for (key, value) in problemtree.items():
-                tree.set(key, value)
+
+        # copy attributes over if not innocufying
+        for (key, value) in problemtree.items():
```

Comment:
Let's remove the comment-- now that the code is not done conditionally, I think it has absolutely no value (except causing us all to wonder if "innocufying" is really a word!).